.github/ISSUE_TEMPLATE/new_package.yml

```diff
@@ -0,0 +1,75 @@
+name: 🆕 New Package
+description: Propose the addition of a new package/tool
+title: <package_name>
+labels: [":new: package"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for contributing to [VM-Packages](https://github.com/mandiant/VM-Packages), your package proposal helps having more available tools for [FLARE-VM](https://github.com/mandiant/flare-vm)! :cupid: Please ensure that the tool doesn't already exist as part of any of the [current packages](https://github.com/mandiant/VM-Packages/tree/main/packages) and that there is no [issue](https://github.com/mandiant/VM-Packages/issues?q=is%3Aopen+is%3Aissue+label%3A%22%3Anew%3A+package%22) already.
+
+        ### Use the package name as the issue title.
+        The convention is to use lowercase names with the following format: `toolname.vm` or `tool-name.vm`. For example `capa.vm` and `fakenet-ng.vm`.
+  - type: input
+    id: name
+    attributes:
+      label: Tool Name
+      description: The name of the tool being installed, normally different from the package name. For example fakenet-ng.vm (package name) vs FakeNet-NG (tool name).
+      placeholder: ex. FakeNet-NG
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    validations:
+      required: true
+    attributes:
+      label: Category
+      description: Which category should this tool be installed to?
+      options:
```

Comment:
We may want to review these (probably not as part of this PR). See https://github.com/mandiant/VM-Packages/issues/38